src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -767,7 +768,12 @@ private async Task CommonStreaming(ConnectionState connectionState, string strea
             catch (OperationCanceledException)
             {
                 Log.CancelingStream(_logger, streamId);
-                responseError = $"Stream canceled by client.";
+                responseError = "Stream canceled by client.";
+            }
+            catch (Exception ex)
+            {
+                Log.ErroredStream(_logger, streamId, ex);
+                responseError = $"Stream errored by client: '{ex.Message}'";
```

Comment:
Why not include the stack trace too?

```suggestion
                responseError = $"Stream errored by client: '{ex}'";
```

Reply:
I don't think it's useful info for the server to have